sdk/loadtestservice/azure-developer-loadtesting/swagger/README.md

```diff
@@ -1,37 +1,70 @@
 java: true
-regenerate-pom: false
+data-plane: true
 title: LoadTestingClient
+package-version: 1.0.0
 security: AADToken
 security-scopes: https://cnt-prod.loadtesting.azure.com/.default
-data-plane: true
-generate-models: false
-generate-samples: false
-generate-tests: false
 artifact-id: azure-developer-loadtesting
 namespace: com.azure.developer.loadtesting
+generate-builder-per-client: false
 partial-update: true
 output-folder: $(azure-sdk-for-java-folder)/sdk/loadtestservice/azure-developer-loadtesting
 service-versions:
-- 2022-06-01-preview
+- '2022-11-01'
 directive:
-- from: swagger-document
-  where: '$.paths["/testruns/{testRunId}"].patch'
-  transform: >
-    $["operationId"] = "TestRun_CreateOrUpdateTestRun";
-- from: swagger-document
-  where: '$.paths.*[?(@.tags=="AppComponent")]'
-  transform: >
-    $["operationId"] = $["operationId"].replace("AppComponent_", "LoadTestAdministration_");
-- from: swagger-document
-  where: '$.paths.*[?(@.tags=="ServerMetrics")]'
-  transform: >
-    $["operationId"] = $["operationId"].replace("ServerMetrics_", "LoadTestAdministration_");
+- from: swagger-document 
+  where: $["paths"]["/tests/{testId}"].patch
+  transform: $["operationId"] = "Test_CreateOrUpdateTest";
+- from: swagger-document 
+  where: $["paths"]["/tests/{testId}"].delete
+  transform: $["operationId"] = "Test_DeleteTest";
+- from: swagger-document 
+  where: $["paths"]["/tests/{testId}"].get
+  transform: $["operationId"] = "Test_GetTest";
+- from: swagger-document 
+  where: $["paths"]["/tests"].get
+  transform: $["operationId"] = "Test_ListTests";
+- from: swagger-document 
+  where: $["paths"]["/tests/{testId}/files/{fileName}"].put
+  transform: $["operationId"] = "Test_UploadTestFile";
+- from: swagger-document 
+  where: $["paths"]["/tests/{testId}/files/{fileName}"].get
+  transform: $["operationId"] = "Test_GetTestFile";
+- from: swagger-document 
+  where: $["paths"]["/tests/{testId}/files/{fileName}"].delete
+  transform: $["operationId"] = "Test_DeleteTestFile";
+- from: swagger-document 
+  where: $["paths"]["/tests/{testId}/files"].get
+  transform: $["operationId"] = "Test_ListTestFiles";
+- from: swagger-document 
+  where: $["paths"]["/test-runs/{testRunId}"].delete
+  transform: $["operationId"] = "TestRun_DeleteTestRun";
+- from: swagger-document 
+  where: $["paths"]["/test-runs/{testRunId}"].patch
+  transform: $["operationId"] = "TestRun_CreateOrUpdateTestRun";
+- from: swagger-document 
+  where: $["paths"]["/test-runs/{testRunId}"].get
+  transform: $["operationId"] = "TestRun_GetTestRun";
+- from: swagger-document 
+  where: $["paths"]["/test-runs/{testRunId}/files/{fileName}"].get
+  transform: $["operationId"] = "TestRun_GetTestRunFile";
+- from: swagger-document 
+  where: $["paths"]["/test-runs"].get
+  transform: $["operationId"] = "TestRun_ListTestRuns";
+- from: swagger-document 
+  where: $["paths"]["/test-runs/{testRunId}:stop"].post
+  transform: $["operationId"] = "TestRun_StopTestRun";
```

Comment:
We should not be doing transforms on the operationIds in the REST API. If the operationIds need to be fixed they should be fixed in the REST API directly.

Reply:
Agree. If this applies to every SDK, it might better to be done in REST API.

One exception I can think of maybe the case that you want Doc site (learning.microsoft.com) display the REST API differently from the client/operation group in SDK.
If so, we can have further discussion.

Reply:
Changed. Corresponding rest-api-specs PR: https://github.com/Azure/azure-rest-api-specs/pull/21672